x-pack/plugins/watcher/public/sections/watch_list/components/watch_list.tsx

```diff
@@ -60,132 +113,227 @@ const WatchListUi = ({ intl }: { intl: InjectedIntl }) => {
-        defaultMessage: 'Last triggered',
-      }),
-      sortable: true,
-      truncateText: true,
-      render: (lastChecked: Moment) => {
-        return lastChecked ? lastChecked.fromNow() : lastChecked;
+      {
+        field: 'watchStatus.comment',
+        name: i18n.translate('xpack.watcher.sections.watchList.watchTable.commentHeader', {
+          defaultMessage: 'Comment',
+        }),
+        sortable: true,
+        truncateText: true,
       },
-    },
-    {
-      actions: [
-        {
-          render: (watch: any) => {
-            return (
-              <EuiButtonEmpty
-                iconType="pencil"
-                isDisabled={watch.isSystemWatch}
-                aria-label={intl.formatMessage({
-                  id: 'xpack.watcher.sections.watchList.watchTable.menuEditButtonDescription',
-                  defaultMessage: 'Edit watch',
-                })}
-                href={`#/management/elasticsearch/watcher/watches/watch/${watch.id}/edit`}
-              >
-                <FormattedMessage
-                  id="xpack.watcher.sections.watchList.watchTable.menuEditButtonTitle"
-                  defaultMessage="Edit"
-                />
-              </EuiButtonEmpty>
-            );
-          },
+      {
+        field: 'watchStatus.lastMetCondition',
+        name: i18n.translate('xpack.watcher.sections.watchList.watchTable.lastFiredHeader', {
+          defaultMessage: 'Last fired',
+        }),
+        sortable: true,
+        truncateText: true,
+        width: '130px',
+        render: (lastMetCondition: Moment) => {
+          return lastMetCondition ? lastMetCondition.fromNow() : lastMetCondition;
         },
-      ],
-    },
-  ];
+      },
+      {
+        field: 'watchStatus.lastChecked',
+        name: i18n.translate('xpack.watcher.sections.watchList.watchTable.lastTriggeredHeader', {
+          defaultMessage: 'Last triggered',
+        }),
+        sortable: true,
+        truncateText: true,
+        width: '130px',
+        render: (lastChecked: Moment) => {
+          return lastChecked ? lastChecked.fromNow() : lastChecked;
+        },
+      },
+      {
+        name: i18n.translate('xpack.watcher.sections.watchList.watchTable.actionHeader', {
+          defaultMessage: 'Actions',
+        }),
+        width: '75px',
+        actions: [
+          {
+            render: (watch: any) => {
+              const label = i18n.translate(
+                'xpack.watcher.sections.watchList.watchTable.actionEditTooltipLabel',
+                { defaultMessage: 'Edit' }
+              );
+              return (
+                <EuiToolTip content={label} delay="long">
+                  <EuiButtonIcon
+                    isDisabled={watch.isSystemWatch}
+                    aria-label={i18n.translate(
+                      'xpack.watcher.sections.watchList.watchTable.actionEditAriaLabel',
+                      {
+                        defaultMessage: 'Edit watch `{name}`',
```

Comment:
Can we change this to:

```js
`Edit watch '{name}'`
```

I'm not sure backticks work as quotation marks in the original code. Perhaps worth checking with Gail.

Reply:
good catch, fixed!